gulpfile.js

```diff
@@ -1,88 +1,112 @@
 var gulp = require('gulp');
-var concatCss = require('gulp-concat-css');
 var sass = require('gulp-sass');
 var minifyCss = require('gulp-minify-css');
 var rename = require('gulp-rename');
-var ts = require('gulp-typescript');
-var es6transpiler = require('gulp-es6-transpiler');
-var jsx = require('gulp-jsx');
+var uglify = require('gulp-uglify');
 var concat = require('gulp-concat');
-var minify = require('gulp-minify');
 var connect = require('gulp-connect');
 var watch =  require('gulp-watch');
+var source = require('vinyl-source-stream');
+var browserify = require('browserify');
+var livereload = require('gulp-livereload');
+var babelify = require('babelify');
+var sourcemaps = require('gulp-sourcemaps');
+var watchify = require('watchify');
+
+
+
 
 /**
  * Specify config for tasks
  */
-var config = {
-    index: 'src/index.html',
-    src: 'src',
-    dist: './dist',
-    distCss: './dist/css',
-    distJs: './dist/js',
-    srcCss: 'src/styles/**/*.sass',
-    srcJs: 'src/js/**/*.ts'
+ var config = {
+  index: 'src/*.html',
+  src: 'src',
+  dist: './dist',
+  distCss: './dist/css',
+  distJs: './dist/js',
+  srcCss: 'src/styles/**/*.sass',
+  srcJs: 'src/js/**/*.jsx, src/js/**/*.js',
+  entryPoint: 'src/js/main.jsx',
+  minifiedOut: 'build.min.js',
+  extensions: ['.js', '.json', '.es6', '.jsx']
 };
 
 /**
- * Copy index file
- */
-gulp.task('copy', function () {
-    gulp.src( config.index )
-        .pipe( gulp.dest( config.dist ) );
-});
-
-/**
  * Start local server for development
  */
-gulp.task('server', ['style', 'scripts', 'copy'], function () {
-
-    /**
-     * Listening port can be specified manually via command `PORT=7777 gulp`
-     * @type {number}
-     */
-    var serverPort = process.env.PORT || 9000;
+ gulp.task('server', ['watch'], function () {
```

Comment:
Do you really need `watch` task as a dependency for `server` task?
```gulp.task('default', ['watch', 'server']);``` invoke `watch` and `server`. `server` will try to invoke `watch` one more time.